### ISSUE TITLE
Documentation issue with default value of padding argument of pad_sequences

### ISSUE THREAD

**Issue by LiquPy** [keras-team/keras-preprocessing#273]
I am not sure if this is the right place to open the issue since the issue is with Keras docs which is in the main repository but it is related to the preprocessing. Anyway, in the documentation under `pad_sequences` at [https://keras.io/preprocessing/sequence/](https://keras.io/preprocessing/sequence/) the default value of `padding` is `'pre'` however, a few lines below it, it says "Sequences that are shorter than `num_timesteps` are padded with `value` at the **end**."
